spec/services/permissions_service_spec.rb

```diff
@@ -1,49 +0,0 @@
-require 'rails_helper'
-
```

Comment:
I assume this means you are working on a new permissions test in place of this one?